app/forms/idv/how_to_verify_form.rb

```diff
@@ -9,8 +9,13 @@ class HowToVerifyForm
 
     attr_reader :selection
 
-    validates :selection,
-              presence: { message: proc { I18n.t('errors.doc_auth.how_to_verify_form') } }
+    validates :selection, presence: {
+      message: proc { I18n.t('errors.doc_auth.how_to_verify_form') },
+    }
+    validates :selection, inclusion: {
+      in: [REMOTE, IPP],
+      message: proc { I18n.t('errors.doc_auth.how_to_verify_form') },
+    }
```

Comment:
`presence` worked as expected. I wanted to ensure we only accepted known values, so I had initially _changed_ it from `presence` to `inclusion`, under the arguably-reasonable expectation that, since `[REMOTE, IPP]` does not include nil, it wouldn't be accepted. Turns out both are needed.